Gulpfile.js

```diff
@@ -252,9 +252,23 @@ function getSentryReleaseVersion() {
     process.env.SENTRY_ENV = "development";
   }
 
-  // If CI, don't use the revision
+  // If CI, don't use the revision since this will either be a real prod release or a manually-
+  // triggered CI build (most likely from a PR that needs more click-testing)
   if (IS_CI) {
-    return "vscode-confluent@" + version;
+    // see https://docs.semaphoreci.com/reference/env-vars#git-branch
+    const upstream = process.env.SEMAPHORE_GIT_BRANCH;
+    const downstream = process.env.SEMAPHORE_GIT_WORKING_BRANCH;
+    console.log(`CI build branches: Upstream: ${upstream}, Downstream: ${downstream}`);
+    // see https://docs.semaphoreci.com/reference/env-vars#pr-number
+    const prNumber = process.env.SEMAPHORE_GIT_PR_NUMBER;
+    console.log(`CI build PR: ${prNumber}`);
+    if (prNumber !== undefined) {
+      // PR build (upstream branch doesn't matter since it isn't a real prod release)
+      return `vscode-confluent@pr${prNumber}-${version}`;
+    } else {
+      // build on main or a release branch
+      return "vscode-confluent@" + version;
```

Comment:
Technically we don't make .vsix files on `main` but I don't know what we'd want to do in that case anyway. We could split this logic and check `downstream === "main"` to add another prefix if necessary.